infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -238,11 +259,17 @@ private static ByteBuffer getByteBuffer(Metadata metadata, AbstractMessage messa
         return buf;
     }
 
-    static ByteBuffer getByteBufferWithMetaData(AbstractMessage message) {
+    @VisibleForTesting
+    public static ByteBuffer getByteBufferWithMetaData(AbstractMessage message) {
         Metadata metadata = getMetadata(message);
         return getByteBuffer(metadata, message);
     }
 
+    @Override
+    public boolean quotaExceeded() {
```

Comment:
The name of quotaExceeded() likes ambiguous. What if usage equals quota?

Reply:
> Exceeded: be greater in number or size than

Please suggest a better name, I can't think of one :)